gradle.properties

```diff
@@ -56,57 +56,57 @@ micronautDocsVersion=1.0.3
 micronautOpenApi=1.1.1
 micronautViewsVersion=1.2.0.RC1
 micronautRssVersion=1.0.0
-junit5Version=5.3.1
+junit5Version=5.3.2
 javaxAnnotationApiVersion=1.3.2
-picocliVersion=3.5.0
+picocliVersion=3.5.2
 reactiveStreamsVersion=1.0.2
 jsr305Version=3.0.2
-caffeineVersion=2.5.2
-yamlVersion=1.23
+caffeineVersion=2.5.6
+yamlVersion=1.24
 asmVersion=7.0
 gormVersion=7.0.1.RELEASE
 gormHibernateVersion=7.0.0.RELEASE
 grailsVersion=3.2.9
-groovyVersion=2.5.6
-jacksonVersion=2.9.8
-lettuceVersion=5.1.3.RELEASE
+groovyVersion=2.5.7
+jacksonVersion=2.9.9
+lettuceVersion=5.1.7.RELEASE
 micronautTestVersion=1.0.4
-nettyVersion=4.1.30.Final
-reactivePgClientVersion=0.11.3
-reactorVersion=3.2.5.RELEASE
-rxJava2Version=2.2.6
-rxJavaInteropVersion=0.13.1
+nettyVersion=4.1.36.Final
```

Comment:
I'd be careful before upgrading Netty because this version includes support for GraalVM. This mean that we need to make sure that their changes are compatible with ours and we don't break anything.